### DISCUSSION TITLE
Ability to multi-download to selected folder

### DISCUSSION THREAD

**Discussion by mosherc** [BlafKing/sd-civitai-browser-plus#139] [Ideas]
Can you add the ability to do multiple-select downloads to a specified sub-folder per download? Ex. let's say I want to download a creator's new clothing LORAs to my /clothes folder, but my default is /uncategorized. It would be nice if I could say for this selection, put these in /clothes and not the default.

**Comment by BlafKing** [comment]
Thanks for the suggestion!

I'll have a look if I can find a good way to implement this, it'll be a bit tricky since it's possible to select multiple different content types, and each content type can have differing sub folders.

I agree that it would be handy to have this feature though, so I'll try to think of a method that works without looking/being too clunky to use :,)

For now, you could change the default sub directory in the settings tab of the extension before downloading as a work around, but that is of course not really an elegant solution.

**Reply by mosherc** [reply]
Yeah this would really only work when you know you are selecting stuff that goes in the same sub folder. If I were searching by creator and selecting multiple Checkpoints, Loras, etc., then you can't really do this and it wouldn't make sense. Thanks!

**Comment by BlafKing** [comment]
Well it took much longer than it should've, but I've implemented this now in v3.3.0! :)
